persistence-jdbc/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/slick/SlickTestEntityReadSide.scala

```diff
@@ -25,7 +26,18 @@ trait Tables {
   }
   lazy val testCounts: TableQuery[TestCounts] = TableQuery[TestCounts]
 
-  def createTable: DBIO[_] = testCounts.schema.create
+  def createTable: DBIO[_] = {
+    // create if exists not exists
+    // Note: this hack will become obsolete in Slick 3.3.0
+    // see https://github.com/slick/slick/pull/1704
+    MTable.getTables.flatMap { tables =>
+      if (!tables.exists(_.name.name == testCounts.baseTableRow.tableName)) {
+        testCounts.schema.create
+      } else {
+        DBIO.successful(())
+      }
+    }.transactionally
+  }
```

Comment:
The test was broken because it tries to create the table over and over again. We didn't see it because it was being skipped.